src/server/acl/acl_family.cc

```diff
@@ -567,6 +568,13 @@ void AclFamily::Init(facade::Listener* main_listener, UserRegistry* registry) {
     return;
   }
   registry_->Init();
+  config_registry.RegisterMutable("aclfile", [this](const absl::CommandLineFlag& flag) {
+    auto res = flag.TryGet<std::string>();
+    if (res.has_value()) {
+      LoadToRegistryFromFile(res.value(), false);
+    }
+    return res.has_value();
```

Comment:
how it is done in redis?

Reply:
That's a good question. When I started working on this feature, I gathered all the available functionality and I did not add any details for this specific case via `CONFIG SET`. I tested it now, and I was very surprised to find out that in redis this config option is `immutable`. I was not aware of this and I am not quite sure how this ended up in my list. 

However, I still think we should move forward with this as it's a good enhancement. I don't see any good reason why we shouldn't allow this, since in general you can edit the file and then call `ACL LOAD` to reload it. What do you say?